docs/rules/fluent-chaining.md

```diff
@@ -0,0 +1,159 @@
+# Enforce formatting for chained calls (fluent-chaining)
+
+This rule enforces consistent spacing and line
+breaks when working with chained expressions.
+
+## Rule Details
+
+This rule aims to provide a consistent format
+when working with chained functions such as
+those provided by lodash or Q promises.
```

Comment:
This paragraph really doesn't say much that wasn't said in the previous paragraph. It's not really "details". Try to some up the rule in two or three clear and simple bullet points here.